### DISCUSSION TITLE
See who Else is checked in

### DISCUSSION THREAD

**Discussion by ChillLP** [Traewelling/traewelling#2011] [Ideas / Feature Requests]
After checking in, you only get a small Windows showing who is checked in, and after that is gone its rather difficult seeing who Else is there. 

I Imagine after pulling Up your own checking, that below that there will be a section of the other people that are on that Ride, a little Bit Like a "you might also Like" Section

I promise i'll be done soon with Feature ideas

**Comment by MrKrisKrisu** [comment]
> I promise i'll be done soon with Feature ideas

Please feel free to open issues for any idea you have. :)

**Comment by pascalwittler** [comment]
The first idea that comes to my mind is to list people who also checked in on the same train above or below the people who liked your status.
